index.html

```diff
@@ -127,7 +127,12 @@ <h2 class="section-title">Add New Task</h2>
               </div>
               <label class="label" for="time">Time:</label>
               <input class="input-time" type="time" id="time" name="time" />
-              <button class="submit-button" name="submit-button" type="button">
+              <button
+                class="submit-button"
+                name="submit-button"
+                type="button"
+                id="submit-button"
```

Comment:
`submit-button` is not a descriptive name. Please use something like `add-task-button` instead.

Reply:
I'll do it. thanks